### DISCUSSION TITLE
Pummel Party（揍击派对）

### DISCUSSION THREAD

**Discussion by YuetKaka15** [pjy612/SteamManifestCache#59] [Share]
App ID: 880940
DecryptionKey && manifest
qhzxyh@163,com
[pummelParty.zip](https://github.com/pjy612/SteamManifestCache/files/13692938/pummelParty.zip)

**Comment by YuetKaka15** [comment]
code：DFB252B91BC51652

**Reply by pjy612** [reply]
查收
